examples/complete/emr-cluster.tf

```diff
@@ -24,9 +25,17 @@ module "emr" {
   # External resource references
   bucket_name_for_root_directory = module.s3-data.bucket_name
   bucket_name_for_logs           = module.s3-logs.bucket_name
-  s3_policy_arns = [
+  additional_policy_arns = [
     module.s3-logs.rw_policy_arn,
-    module.s3-data.rw_policy_arn
+    module.s3-data.rw_policy_arn,
+    "arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy"
+  ]
+  bootstrap_actions = [
```

Comment:
Something for the backlog would be adding the log4j bootstrap actions to these examples:
https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-log4j-vulnerability.html

Reply:
Sure!